app.go

```diff
@@ -76,7 +76,7 @@ func (app *App) Run() {
 		cron := cron.New()
 		_, _ = cron.AddFunc(app.config.schedule, func() {
 			// make sure we have the latest translations
-			app.translations.Fetch()
+			hashes := app.translations.Fetch()
```

Comment:
This is useless as the `hashes` variable is a pointer to the map containing the hashes. I guess it would be better to use a copy to avoid concurrent map access issues...

Reply:
Here, the `hashes` is scoped to the anonymous `func` (I used `:=`) so we should not have any concurrences

Reply:
But ok, I see. The returned `hashes` are a also stored into the refresher and we reuse this pointer again.

Reply:
Yes, but even if you define an other variable, you are still using the same map in memory because `app.translations.Fetch()` always returns a pointer to the same map instance. See https://github.com/graillus/translations-refresher/blob/main/loco.go#L36

Reply:
> Ok, I see. The returned `hashes` are a also stored into the refresher and we reuse this pointer again. There is a reason why we store the hash map internal instead of generating one every time we call `translations.Fetch` ?

I don't see any reason not to return a new instance

Reply:
Because this is not a "fix", I removed the commit (useless and we can think that we've "fixed" something, that is not the case)